onnxscript/rewriter/pattern.py

```diff
@@ -1197,6 +1201,7 @@ def match(
         graph_or_function: ir.Graph | ir.Function,
         node: ir.Node,
         verbose: int = 0,
+        remove_nodes: bool = True,
```

Comment:
Helpful to document new args?

Reply:
Good point. Will do this separately, there are likely to be overlapping (merge) conflicts from other changes I have been making concurrently to extend pattern matcher.